### PR TITLE
docs(cli): remove API v1 refs and add examples

### PR DESCRIPTION
**Pull request by J-C-B**
Update docs for `lacework api` command.

### CODE REVIEW

File: cli/cmd/api.go [hunk 1]
```diff
@@ -41,22 +41,20 @@ var (
 		Short: "Helper to call Lacework's API",
 		Long: `Use this command as a helper to call any available Lacework API v1 & v2 endpoint.
```
Comment (afiune) [inline cli/cmd/api.go:42]:
```suggestion
		Long: `Use this command as a helper to call any available Lacework API v2 endpoint.
```

File: cli/cmd/api.go [hunk 2]
```diff
@@ -41,22 +41,20 @@ var (
...
-    https://<ACCOUNT>.lacework.net/api/v1/external/docs
-
-### For APIv2
-
-To list all available Lacework schema types:
+To list all available Lacework schema type:
 
     lacework api get /v2/schemas
 
+Example usage,
```
Comment (afiune) [inline cli/cmd/api.go:51]:
```suggestion
```

File: cli/cmd/api.go [hunk 3]
```diff
@@ -41,22 +41,20 @@ var (
...
+	To recieve a json response of all machines within the given time window
+	lacework api post /api/v2/Entities/Machines/search -d "{\"timeFilter\":{\"startTime\":\"2023-05-10T00:00:00Z\",\"endTime\":\"2023-05-14T00:00:00Z\"}}"   
+	
+	To recieve a json response of all agents within the given time window
```
Comment (afiune) [inline cli/cmd/api.go:55]:
```suggestion
To receive a json response of all agents within the given time window
```

File: cli/cmd/api.go [hunk 4]
```diff
@@ -41,22 +41,20 @@ var (
 		Short: "Helper to call Lacework's API",
 		Long: `Use this command as a helper to call any available Lacework API v1 & v2 endpoint.
 
-### For APIv1
 
-To list all integrations configured in your account:
+### APIv2
```
Comment (afiune) [inline cli/cmd/api.go:45]:
```suggestion
### API v2
```

File: cli/cmd/api.go [hunk 5]
```diff
@@ -39,24 +39,21 @@ var (
...
+To list all available Lacework schema type:
 
     lacework api get /v2/schemas
 
+To receive a json response of all machines within the given time window
+	lacework api post /api/v2/Entities/Machines/search -d "{}"   
+	
+To receive a json response of all agents within the given time window
+	lacework api post /api/v2/AgentInfo/search -d "{}"  
+
```
Comment (afiune) [inline cli/cmd/api.go:56]:
```suggestion
To receive a json response of all machines within the given time window:

	lacework api post /api/v2/Entities/Machines/search -d "{}"   
	
To receive a json response of all agents within the given time window:

	lacework api post /api/v2/AgentInfo/search -d "{}"  

```

File: cli/cmd/api.go [hunk 6]
```diff
@@ -39,23 +39,22 @@ var (
...
-### For APIv2
+To receive a json response of all machines within the given time window:
 
-To list all available Lacework schema types:
+	lacework api post /api/v2/Entities/Machines/search -d "{}"   
+	
+To receive a json response of all agents within the given time window:
 
-    lacework api get /v2/schemas
+	lacework api post /api/v2/AgentInfo/search -d "{}"
```
Comment (afiune) [inline cli/cmd/api.go:57]:
```suggestion
    lacework api post /api/v2/AgentInfo/search -d "{}"  
```

**Review by afiune — CHANGES_REQUESTED** [review]
You have to also update this test resource to see how this renders

https://github.com/lacework/go-sdk/blob/main/integration/test_resources/help/api

File: cli/cmd/api.go [hunk 7]
```diff
@@ -41,22 +41,20 @@ var (
...
-    lacework api get /v1/external/integrations
-
-For a complete list of available API v1 endpoints visit:
-
-    https://<ACCOUNT>.lacework.net/api/v1/external/docs
-
-### For APIv2
-
-To list all available Lacework schema types:
+To list all available Lacework schema type:
```
[the remark this answers is not in the document]
Comment (lacework-aaronscheiber) [reply cli/cmd/api.go:47]: types

File: cli/cmd/api.go [hunk 8]
```diff
@@ -39,23 +39,22 @@ var (
...
-		Long: `Use this command as a helper to call any available Lacework API v1 & v2 endpoint.
+		Long: `Use this command as a helper to call any available Lacework API v2 endpoint.
 
-### For APIv1
 
-To list all integrations configured in your account:
+### API v2
 
-    lacework api get /v1/external/integrations
+To list all available Lacework schema type:
```
Comment (lacework-aaronscheiber) [inline cli/cmd/api.go:47]:
```suggestion
To list all available Lacework schema types:
```

**Review by lacework-aaronscheiber — COMMENTED** [review]
otherwise lgtm

**Review by afiune — APPROVED** [review]

**Comment by afiune** [comment]
make it so

**Pull request merged**